README.md

```diff
@@ -50,7 +50,7 @@ Please also make sure `HADOOP_HOME`, `LD_LIBRARY_PATH`, `CLASSPATH` is set corre
 ```shell
 export HADOOP_HOME=/path/to/hadoop
 export LD_LIBRARY_PATH=${JAVA_HOME}/lib/server:${LD_LIBRARY_PATH}
-export CLASSPATH=$(find $HADOOP_HOME -iname "*.jar" | xargs echo | tr ' ' ':')
+export CLASSPATH=$(${HADOOP_HOME}/bin/hadoop classpath --glob)
```

Comment:
In some use cases, `hadoop classpath --glob` is not correct. Maybe we can add this as an alternative?

Reply:
Sorry, could you help indicate which case will be wrong? I have use this way in another project, which has been running in several years. But I have to say I always use this in CDH release.